### ISSUE TITLE
Zjednodusenie flow pre nahlasovanie a vyber miesta

### ISSUE THREAD

**Issue by jsuchal** [psekan/somvrade#6]
Rozmyslal som ako toto zjednodusit:

Pre hlasenie:

Ked pridem na miesto, tak len nahlasit pocet ludi. Neviem ci bude pre ludi velka motivacia po X hodinach tam pisat, ze kolko tam cakali. Resp oddelit hlasenie poctu a dlzky cakania ako separe flowy.

Asi dava zmysel aby to prebiehalo nejako takto:
- vyber miesta
- button chcem nahlasit pocet ludi
- button chcem nahlasit dlzku cakania (tu asi treba dat moznost zadat cas kedy to bolo)

Pre vyber miesta:
- ak sa rozhodujem na ake miesto chcem ist, tak by som chcel asi vidiet tabulku pre cely okres a nie si to preklikavat po jednom.

Super napad toto.

**Comment by sharkix** [comment]
Este mozno stoji za zmienku ako oznacit konkretne odberne miesto, ked su vacsinou po dve a viac. 
Niekde su drive-thru, niekde externe aj interne... 
Ako spravne urcit v ktorom rade stojim, ked ich moze byt viac?

**Comment by psekan** [comment]
@jsuchal Áno, výber miesta bude prebiehať len raz a potom sa užívateľ rozhodne, či ide nahlásiť počet čakajúcich. Ideálne chcem docieliť, že sa po zadaní počtu čakajúcich zobrazí časovač, ako dlho v rade čakáte, aj s možnosťou nahlásenia ukončenia čakania. Len treba docieliť, aby nenahlasovali ukončenie skôr, ako reálne skončia. 

Do soboty nestihnem pripraviť zobrazenie všetkých miest v okrese, aby sa rozhodli ľudia, kde je v danom čase ideálne ísť.

@sharkix Bohužiaľ, toto neviem ešte dobre vyriešiť. Jediné riešenie je označiť tie miesta čo najlepšie a zároveň ak sa nedajú odlíšiť, uvádzať počet za všetky rady spolu. Za akýkoľvek návrh/nápad budem vďačný.

**Comment by janfratric** [comment]
pri zobrazovani sa mozete inspirovat tu:
https://dubravka.korona-test.sk/
su tam viacere odberne miesta na jednej adrese.

zadavanie dat je riesene uplne inak, ale vizualizacia stoji za uvahu
